.evergreen/start-atlas-cloud-cluster.sh

```diff
@@ -46,6 +46,10 @@
 #
 #   (ATLAS_CLOUD_TEST_CLUSTER_NAME="TestCluster" source .evergreen/start-atlas-cloud-cluster.sh \
 #     && npm run -w compass-e2e-tests test web -- --test-atlas-cloud-sandbox --test-filter="atlas-cloud/**/*")
+#
+# When setting up for the first time, make sure you:
+# - Add payment details to be able to create clusters. You can use stripe test card.
+# - Allow network access to the project for your IP address.
```

Comment:
Hah, just hit me that we can make this step unnecessary also with atlas cli ([in this PR](https://github.com/mongodb-js/compass/pull/6432))